### PR TITLE
Added a task and messages to handle synching code_safety and deleted …

### PR DESCRIPTION
**Pull request by bpellegr**
…files

### CODE REVIEW

File: lib/tasks/audit_code.rake [hunk 1]
```diff
@@ -143,7 +151,7 @@ def flag_file_as_safe(release, reviewed_by, comments, f)
   safety_cfg = YAML.load_file(SAFETY_FILE)
   file_safety = safety_cfg['file safety']
 
-  unless File.exist?(f)
+  unless File.exist?(f)
```
Comment (joshpencheon) [inline lib/tasks/audit_code.rake]: trailing whitespace

File: lib/tasks/audit_code.rake [hunk 2]
```diff
@@ -448,6 +472,17 @@ Usage:
     flag_file_as_safe(release, ENV['reviewed_by'], ENV['comments'], ENV['file'])
   end
 
+  desc 'Deletes any files from code_safety.yml that are no longer in repository.'
+  task(:tidy_code_safety_file) do
+#    abort('You have local changes, cannot verify code safety!') unless clean_working_copy?
+
+    puts 'Checking code safety...'
+
+    begin
```
Comment (joshpencheon) [inline lib/tasks/audit_code.rake]: redundant block

File: lib/tasks/audit_code.rake [hunk 3]
```diff
@@ -448,6 +472,17 @@ Usage:
     flag_file_as_safe(release, ENV['reviewed_by'], ENV['comments'], ENV['file'])
   end
 
+  desc 'Deletes any files from code_safety.yml that are no longer in repository.'
+  task(:tidy_code_safety_file) do
+#    abort('You have local changes, cannot verify code safety!') unless clean_working_copy?
```
Comment (joshpencheon) [inline lib/tasks/audit_code.rake]: agree we don't need this, can remove comment

File: lib/tasks/audit_code.rake [hunk 4]
```diff
@@ -396,6 +405,21 @@ def clean_working_copy?
  end
 end
 
+def remove_non_existent_files_from_code_safety
+  safety_cfg = YAML.load_file(SAFETY_FILE)
+  file_safety = safety_cfg['file safety']
+  files_no_longer_in_repo = file_safety.keys.select {|ff| !File.file?(ff)}
```
Comment (joshpencheon) [inline lib/tasks/audit_code.rake]: Can use `reject` here instead

**Comment by timgentry** [comment]
As a heads up @bpellegr, rubocop would catch many of Josh's suggestions, including the reject I _think_

**Pull request merged**